File: Learnify.Domain/Entities/Review.cs
namespace Learnify.Domain.Entities
{
    public class Review : BaseEntity
    {
        public double Rating { get; set; }
        public string Comment { get; set; } = string.Empty;
        public DateTime ReviewdOn { get; set; } = DateTime.UtcNow;
        public Guid UserId { get; set; }
        public User? User { get; set; }
        public Guid CourseId { get; set; }
        public Course? Course { get; set; }
    }
}

File: Learnify.UI/src/models/entities/Review.js
export class Review {
    constructor(
        id,
        rating,
        comment,
        reviewedOn = new Date(),
        userId,
        user = null,
        courseId,
        course = null
    ) {
        this.id = id;
        this.rating = rating;
        this.comment = comment;
        this.reviewedOn = reviewedOn;
        this.userId = userId;
        this.user = user;
        this.courseId = courseId;
        this.course = course;
        
    }
}